lua/nvim-treesitter/indent.lua

```diff
@@ -40,7 +40,7 @@ function M.get_indent(lnum)
   local parser = parsers.get_parser()
   if not parser or not lnum then return -1 end
 
-  local node = get_node_at_line(parser:parse():root(), lnum-1)
+  local node = get_node_at_line(parser:parse()[1]:root(), lnum-1)
```

Comment:
cool so i can use the language tree for indent ?

Reply:
I think so, but that'll need some more work beforehand